Python/Product/PythonTools/PythonTools/Intellisense/NormalCompletionAnalysis.cs

```diff
@@ -136,7 +137,7 @@ internal NormalCompletionAnalysis(VsProjectAnalyzer analyzer, ITextSnapshot snap
                     }
                 }
 
-                if (pyReplEval != null && _snapshot.TextBuffer.GetAnalyzer(_serviceProvider).ShouldEvaluateForCompletion(text)) {
+                if (pyReplEval != null && pyReplEval.ReplAnalyzer.ShouldEvaluateForCompletion(text)) {
```

Comment:
Should `ReplAnalyzer` and `_analyzer` be identical here? Should we assert that somewhere?